modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -33,15 +33,56 @@ This {release} version of the software and programming language should not be us
 
 == Highlights of what's new in {release}
 
-The {release} release includes both user-visible new features and internal enhancements and fixes. The most significant new features and capabilities include the following:
+The {release} release includes both user-visible new features and internal enhancements and fixes. The most significant new features and fixes include the following:
+
+* You can now access the {IC} network running remotely without user name and password credentials using the new `+ic+` network alias.
++
+In previous releases, the network alias `+tungsten+` was pre-configured in the project `+dfx.json+` file.
+With authorized credentials, you could then use this network alias to access to the {IC} network when running `+dfx+` commands by using the `+--network=tungsten+` option.
```

Comment:
Shouldn't `tungsten` be `ic` here?

Reply:
One paragraph is about the previous state of affairs (using tungsten) and one paragraph is about the new state of affairs (ic), so that (hopefully) people who had been on tungsten see the difference.

BTW, I was expecting the Random.mo module to be in either 0.6.11 or 0.6.12, but it doesn't seem like it has been picked up. I just installed 0.6.12, for example, and ExperimentalFunds.mo is there but Random.mo isn't:
```
-r-x------   1 lisagunn  staff    811 Dec 31  1969 Prelude.mo
-r-x------   1 lisagunn  staff   1332 Dec 31  1969 Principal.mo
-r-x------   1 lisagunn  staff   5815 Dec 31  1969 RBTree.mo
-r-x------   1 lisagunn  staff   4325 Dec 31  1969 Result.mo
-r-x------   1 lisagunn  staff    886 Dec 31  1969 Stack.mo
```
It seems like it should have been picked up by now. Maybe this is worth looking into?

Reply:
@chenyan-dfinity `Random.mo` is on both the `master` and `next-moc` branches. Maybe the `dfx` build didn't bump the base lib?

Reply:
It was merged in motoko repo today: https://github.com/dfinity/motoko/pull/2044. Probably because mergify was not working last week.